src/main/java/org/folio/circulation/resources/MarkItemMissingResource.java

```diff
@@ -0,0 +1,55 @@
+package org.folio.circulation.resources;
+
+import static org.folio.circulation.domain.validation.NotInItemStatusValidator.refuseWhenItemIsNotClaimedReturned;
+import static org.folio.circulation.support.Result.failed;
+import static org.folio.circulation.support.Result.succeeded;
+import static org.folio.circulation.support.ValidationErrorFailure.singleValidationError;
+
+import io.vertx.core.http.HttpClient;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.Router;
+import io.vertx.ext.web.RoutingContext;
+import org.folio.circulation.domain.Loan;
+import org.folio.circulation.domain.representations.ChangeItemStatusRequest;
+import org.folio.circulation.support.Result;
+
+import java.util.concurrent.CompletableFuture;
+
+public class MarkItemMissingResource extends ChangeStatusResource {
+  public static final String COMMENT = "comment";
+
+  public MarkItemMissingResource(HttpClient client) {
+    super(client);
+  }
+
+  @Override
+  public void register(Router router) {
+    router.post("/circulation/loans/:id/mark-item-missing")
+      .handler(this::changeItemStatus);
+  }
+
+  protected CompletableFuture<Result<Loan>> validate(Result<Loan> loanResult){
+    return super.validate(loanResult)
+      .thenApply(loan-> refuseWhenItemIsNotClaimedReturned(loan));
```

Comment:
Are we able to mark as missing only claimed returned items? If so, then i think we have to rename API to describe it.

Reply:
Yes, we are. How do you suggest renaming API?

Reply:
I'm happy for us to make the intent of the API more explicit. I think we need to be careful that we don't couple the external API design too closely with the internal logic.

Before renaming this API, please can we check with Emma that it is likely that it will always be the case that only loans for items that are claimed returned can be marked missing?

Reply:
Confirmed by Emma

Reply:
Thanks @Svitlana-Zmiivska! 

I'd use something like `resolve-claim-as-missing` or similar.